bin/tape

```diff
@@ -26,10 +26,13 @@ opts.require.forEach(function(module) {
 });
 
 opts._.forEach(function (arg) {
-    glob(arg, function (err, files) {
-        files.forEach(function (file) {
-            require(resolvePath(cwd, file));
-        });
+    var files;
+    try {
+        files = glob.sync(arg);
```

Comment:
> When in the async version would an error be generated by glob, compared to the sync version?

I expect would be generated in the same situations. The docs are unclear about which conditions generate errors. In the case that a glob matches no files, an empty array of files is returned (no error).

I agree with you about `undefined.forEach` being unhelpful. I will update the PR so that `glob.sync` throws. An alternative would be to suppress/ignore any errors (as the original code was doing).